go/cmd/vtctldclient/command/vreplication/vdiff/vdiff.go

```diff
@@ -438,7 +408,7 @@ HasMismatch:  {{.HasMismatch}}
 StartedAt:    {{.StartedAt}}
 {{if (eq .State "started")}}Progress:     {{printf "%.2f" .Progress.Percentage}}%%{{if .Progress.ETA}}, ETA: {{.Progress.ETA}}{{end}}{{end}}
 {{if .CompletedAt}}CompletedAt:  {{.CompletedAt}}{{end}}
-{{range $table := .TableSummaryMap}} 
+{{range $table := .TableSummaryMap}}
```

Comment:
Small bit of trailing whitespace here and on line 420. I suspect it's not load bearing but if it turns out to be, I can undo this.